backend/stakepoold/rpc/api.proto

```diff
@@ -40,6 +41,15 @@ message GetLiveTicketsResponse {
 	repeated Ticket tickets = 1;
 }
 
+message GetTicketInfoRequest {
+    string Hash = 1;
+}
+message GetTicketInfoResponse {
+	string MultiSigAddress = 1;
+	string VspFeeAddress = 2;
+	string OwnerFeeAddress = 3;
```

Comment:
Is there a reason we can't use the user's half of the multisig address for verification? That would cut down on the infomation we need to store, as I don't think we are concerned about the user's reward address anywhere else in dcrstakepool, correct me if I'm wrong.

Reply:
That'd require that the user store the original address used in generating the multisig. If the user loses that information, perhaps because of restoring his wallet on a new device, he may not be able to determine that address to use in making API requests. The user's reward address on the other hand can be quite easily determined given a ticket hash.

Reply:
Ah, I see.

Reply:
If we are going to use this address, I kinda think the name doesn't fit. It's not a fee but a reward for staking, and who is the owner, the vsp owner? What do you think about changing the name to "UserRewardAddress" ? When I read it now, I think it is an address for a fee we are taking.

Also, this assumes the user knows the txid.  What exactly are the conditions in which a wallet restored from seed would know or not know about a particular ticket?

Reply:
`UserRewardAddress` is a finer name.

> Also, this assumes the user knows the txid. What exactly are the conditions in which a wallet restored from seed would know or not know about a particular ticket?

@jrick answered this in riot https://matrix.to/#/!HEeJkbPRpAqgAwhXWO:decred.org/$1566480764154gfDiC:zettaport.com?via=decred.org&via=matrix.org&via=pintobyte.com, `depends on if commitments or change were also assigned to the {user's} wallet`, which is the case currently.